packages/ckeditor5-emoji/src/ui/emojipickerview.ts

```diff
@@ -38,49 +44,49 @@ export default class EmojiPickerView extends View<HTMLDivElement> {
 	/**
 	 * Helps cycling over focusable {@link #items} in the view.
 	 */
-	protected readonly _focusCycler: FocusCycler;
+	public readonly focusCycler: FocusCycler;
 
 	/**
 	 * An instance of the `EmojiSearchView`.
 	 */
-	public searchView: EmojiSearchView;
+	public readonly searchView: EmojiSearchView;
 
 	/**
 	 * An instance of the `EmojiToneView`.
 	 */
-	public toneView: EmojiToneView;
+	public readonly toneView: EmojiToneView;
 
 	/**
 	 * An instance of the `EmojiCategoriesView`.
 	 */
-	public categoriesView: EmojiCategoriesView;
+	public readonly categoriesView: EmojiCategoriesView;
 
 	/**
 	 * An instance of the `EmojiGridView`.
 	 */
-	public gridView: EmojiGridView;
+	public readonly gridView: EmojiGridView;
 
 	/**
-	 * An instance of the `EmojiInfoView`.
+	 * An instance of the `EmojiGridView`.
 	 */
-	public infoView: EmojiInfoView;
+	public readonly resultsView: SearchInfoView;
 
 	/**
 	 * @inheritDoc
 	 */
-	constructor( locale: Locale, dropdownPanelContent: DropdownPanelContent ) {
+	constructor( locale: Locale, dropdownPanelContent: EmojiDropdownPanelContent ) {
```

Comment:
This class should take over lots of responsibilities from the controller (plugin). See my comments in the `EmojiPicker` plugin to learn more.